autocorrelation.py

```diff
@@ -64,7 +90,7 @@ def autocorrelation(input_stream: Stream, atom_keys: Collection, distance: int =
             GenericFilter(
                 inputs="db_tuples",
                 outputs="db_atoms",
-                operation=lambda element: element[0]
+                operation=lambda element: element[1]
```

Comment:
Wait what's the element[0] then? `ID`? Are we sure it's ALWAYS `ID` first?

Extra: Is there a way to use `element['data_json']`?

Reply:
I think I can change to "data_json" and the output should be the same

Reply:
I forgot our streams use a cursor instead of sqlalchemy stuff (was necessary or I could not figure out how to put the results in our Stream) so we always have a tuple, it's always going to be ID first because that is how the table is defined